app/viewModels/transportEquipment/index/seals/SealIdentificationNumberViewModel.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels.transportEquipment.index.seals
+
+import models.{CheckMode, Mode, NormalMode}
+import play.api.i18n.Messages
+
+import javax.inject.Inject
+
+case class SealIdentificationNumberViewModel(heading: String, title: String, requiredError: String)
+
+object SealIdentificationNumberViewModel {
+
+  class SealIdentificationNumberViewModelProvider @Inject() () {
```

Comment:
I think you can use @kenneth-rayner common trait: app/viewModels/ModeViewModelProvider.scala.
Add some overloaded methods in there as you don't have the index.